Add support for MSI GP76 11UX - E17K3IMS.322

NamitBhutani opened the pull request.

glpnk commented:
Have you tested this? it will not work

NamitBhutani commented:
> Have you tested this? it will not work

What do you think is the issue? Will test it soon.

glpnk commented:
You getting confused with BIOS and EC version. Please test if your code compiles before pushing. You can amend first commit and fix code tabulation.

NamitBhutani commented:
> have you tested module with your patch? i've squashed all commits into one and fixed something

Yes, tested it MControlCenter, changed settings and hex dumped.
